IronSoftware.Drawing/IronSoftware.Drawing.Common/AnyBitmap.cs

```diff
@@ -2281,6 +2287,29 @@ private void OpenTiffToImageSharp(ReadOnlySpan<byte> bytes)
             }
         }
 
+        private Compression SetTiffCompression(Tiff tif)
```

Comment:
Is this applicable for the other `OpenTiffToXXX` methods?

Reply:
Not applicable for other because we used only `SixLabors`.